### DISCUSSION TITLE
osmand live menu is lost in 4.0

### DISCUSSION THREAD

**Discussion by pebogufi** [osmandapp/OsmAnd#11226] [Q&A] [answered]
### Description
The menu "OsmAnd Live" is lost in latest versions, so I can't select regions for live update anymore.

### Your Environment
OsmAnd Version: OsmAnd~  4.0.0#31686m.apk, 2021-03-17
Android version: 10
Device model: samsung tab S5e

**Comment by vshcherb** [comment, marked as answer]
OsmAnd Live menu is split in 2 categories now: 
- Purchases: Settings -> Purchases
- Downloads: Download Maps -> Updates -> OsmAnd Live

**Reply by pebogufi** [reply]
The new download layout is very smart and the logic where it is located also.
